src/Discovery/DiscoveryListView.tsx

```diff
@@ -72,19 +72,16 @@ const DiscoveryListView: React.FunctionComponent<Props> = (props: Props) => {
           if (props.config.features.authorization.enabled) {
             disabled = (record[props.accessibleFieldName] !== AccessLevel.ACCESSIBLE) && (record[props.accessibleFieldName] !== AccessLevel.MIXED);
           }
-          // If enableFillRequestForm is true, allow users to check the checkbox without login
-          if (props.config.features.exportToWorkspace?.enableFillRequestForm) {
-            disabled = false;
-          }
-          // Check if the fillRequestFormCheckField is empty, and disable the checkbox if true
           const fillRequestFormCheckField = props.config.features.exportToWorkspace?.fillRequestFormCheckField;
-          if (fillRequestFormCheckField) {
-            const fieldValue = record[fillRequestFormCheckField];
-            if (!fieldValue || fieldValue.length === 0) {
-              disabled = true;
-            }
+          const enableFillRequestForm = props.config.features.exportToWorkspace?.enableFillRequestForm;
+
+          if (!enableFillRequestForm || (!record[fillRequestFormCheckField]?.length)) {
+            disabled = true;
```

Comment:
I don't think this is right. `disabled` is defaulted to a falsy value, so you need to very careful when setting it explicitly to true
I believe the way you write currently will disable all the checkboxes if the `enableFillRequestForm` is not set in config

Reply:
I did. please check now

Reply:
I see what you are trying to say, this can create confusion, I have added a new property variable `enableCheckboxWithoutLogin` which will allow the discovery portal to enable checkboxes without logging in. is this a better solution?

Reply:
No you don't need this extra field in config. What you have before is already doing this because of https://github.com/urvi-occ/data-portal/blob/df82da937789dd1215e3e403dc7ded8e1fd9a972/src/Discovery/DiscoveryListView.tsx#L76-L78
Sinply copy and paste this blcok https://github.com/urvi-occ/data-portal/blob/df82da937789dd1215e3e403dc7ded8e1fd9a972/src/Discovery/DiscoveryListView.tsx#L79-L86 into the if https://github.com/urvi-occ/data-portal/blob/df82da937789dd1215e3e403dc7ded8e1fd9a972/src/Discovery/DiscoveryListView.tsx#L76-L78 would do it, or just revert back to this commit https://github.com/uc-cdis/data-portal/pull/1551/commits/df82da937789dd1215e3e403dc7ded8e1fd9a972

Reply:
done